packages/browser/src/tracing/browserTracingIntegration.ts

```diff
@@ -206,6 +216,12 @@ export const browserTracingIntegration = ((_options: Partial<BrowserTracingOptio
   if (enableLongTask) {
     startTrackingLongTasks();
   }
+
+  // At most one of long-task and long-animation-frames should be enabled, since they track the same events
+  if (!enableLongTask && enableLongAnimationFrame) {
+    startTrackingLongAnimationFrames();
```

Comment:
If we are tracking long animation frames we should not be tracking long tasks, long animation frames basically replace long tasks right? Don't they just overlap on the trace view?

What do you think about something like so?

```js
// browserSupportsLongAnimationFrame is a function that
// validates if browser supports long animation frame
if (enableLongAnimationFrame && browserSupportsLongAnimationFrame()) {
  startTrackingLongAnimationFrames();
} else if (enableLongTask) {
  startTrackingLongTasks();
}
```